pkg/parser/threefour/parser.go

```diff
@@ -305,6 +315,8 @@ func decodeVerdict(eventType uint8) pb.Verdict {
 		return pb.Verdict_DROPPED
 	case monitorAPI.MessageTypeTrace:
 		return pb.Verdict_FORWARDED
+	case monitorAPI.MessageTypePolicyVerdict:
+		return pb.Verdict_FORWARDED
```

Comment:
trying to understand this part. so the verdict field of the verdict message is always forwarded? :)

Reply:
Yes. The naming might be a bit confusing. But a verdict event is only ever emitted if there is a policy that matches the flow. If there is not policy event, then the flow will be dropped.

Reply:
I was wrong. Adapted the code to return the proper verdict.